api/src/main/java/marquez/db/JobDao.java

```diff
@@ -244,10 +242,14 @@ job_tags as (
           ON f.run_uuid = jv.latest_run_uuid
         LEFT OUTER JOIN job_tags jt
           ON j.uuid  = jt.uuid
+        LEFT JOIN runs r
+          ON r.uuid = jv.latest_run_uuid
+        WHERE
+            r.current_run_state ilike :lastRunState OR r IS NULL
```

Comment:
Minor: We want to match on the enum run state using `r.current_run_state = :lastRunState `.